packages/valist-sdk/src/typesApi.ts

```diff
@@ -48,4 +48,9 @@ export interface Listing {
   project_meta: ProjectMetaApi;
   project_name: string;
   timestamp: number;
+  isVerified?: boolean;
+  isMetaMaskVerified?: boolean;
+  isMetMaskCompatible?: boolean;
```

Comment:
```suggestion
  isMetaMaskMarketplace?: boolean;
```

Reply:
This change would need to be applied as well on hp-store, but atm it is fine to be pushed as it is optional 🙌